internal/proto/query_coord.proto

```diff
@@ -203,6 +203,7 @@ message SegmentLoadInfo {
 
 message FieldIndexInfo {
   int64 fieldID =1;
+  // deprecated
```

Comment:
why deprecate this ?

Reply:
`estimateSegmentSize` located in querycoord/util.go is still using this

Reply:
The modification of queyrcoord.proto should be cautious, which may cause some incompatibilities.